tools/labs/skels/kernel_modules/5-oops-mod/oops_mod.c

```diff
@@ -0,0 +1,27 @@
+#include <linux/module.h>
```

Comment:
What generated the oops in this file? (just using this file to ask the question)

Reply:
Trying to access and change memory address 0